core/include/mm/tee_mmu_types.h

```diff
@@ -29,34 +29,47 @@
 #define TEE_MMU_TYPES_H
 
 #include <stdint.h>
+#include <sys/queue.h>
+#include <util.h>
 
-#define TEE_MATTR_VALID_BLOCK		(1 << 0)
-#define TEE_MATTR_HIDDEN_BLOCK		(1 << 1)
-#define TEE_MATTR_HIDDEN_DIRTY_BLOCK	(1 << 2)
-#define	TEE_MATTR_TABLE			(1 << 3)
-#define	TEE_MATTR_PR			(1 << 4)
-#define	TEE_MATTR_PW			(1 << 5)
-#define	TEE_MATTR_PX			(1 << 6)
+#define TEE_MATTR_VALID_BLOCK		BIT(0)
+#define TEE_MATTR_HIDDEN_BLOCK		BIT(1)
+#define TEE_MATTR_HIDDEN_DIRTY_BLOCK	BIT(2)
+#define	TEE_MATTR_TABLE			BIT(3)
```

Comment:
minor: get rid of the few tabs between `#define` and the macro label?

Reply:
OK